bundles/GlossaryBundle/src/Migrations/Version20221215071650.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\GlossaryBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Bundle\GlossaryBundle\GlossaryBundle;
+use Pimcore\Db;
+use Pimcore\Model\Tool\SettingsStore;
+
+/**
+ * Checking if glossary tables already exist
+ */
+final class Version20221215071650 extends AbstractMigration
```

Comment:
I guess this needs to be moved to Core migrations otherwise we get into chicken-egg problem as the bundle needs to be installed to run the migration and the migration needs to be executed to install the bundle.